gulpfile.js

```diff
@@ -202,6 +203,11 @@ gulp.task('imagemin', function () {
     .pipe(gulp.dest('public/dist/img'));
 });
 
+// Make sure upload directory exists
+gulp.task('mkdir:upload', function () {
+  return fs.mkdir('modules/users/client/img/profile/uploads', function () { });
```

Comment:
log it or something maybe? We basically need to notify users of it at the least.